src/libumf_noOS.def

```diff
@@ -0,0 +1,47 @@
+;;;; Begin Copyright Notice
```

Comment:
Perhaps libumf_no_os_provider would be a better name for this file. The current name suggests "no operating system" which is confusing. It's not clear that OS refers to the kind of a provider.

Reply:
Done